ItzCrazyKns/Perplexica
# feat(server): replace existing search functionality with searxng library

This commit replaces the existing search functionality with the Searxng library.

It solves the limitations and issues present in the previous search implementation by leveraging the advanced capabilities of the Searxng library.

## How to test

- Ensure the Searxng service is running at http://127.0.0.1:8080/search.
- Start your application that integrates the new search functionality.
- Perform various search queries through the application interface to test the new functionality.
- Check that the search results are accurately returned from the Searxng service.
- Ensure the results are correctly formatted and displayed in the application.
